src/pages/blog/components/_markets-news.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { SectionContainer, Container, Flex } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { LocalizedLink } from 'components/localization'
+import EyeIcon from 'images/svg/eye.svg'
+import device from 'themes/device'
+
+const MarketsNewsWrapper = styled.div`
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-gap: 30px;
+    padding: 10px;
+    max-height: 211px;
+
+    @media ${device.tabletL} {
+        grid-template-columns: auto auto;
+        max-height: 330px;
+    }
+
+    @media ${device.tabletS} {
+        grid-template-columns: auto;
+        margin: 0 auto;
+        max-height: 668px;
+    }
+`
+
+const StyledSpan = styled.span`
+    background-color: var(--color-blue-10);
+    padding: 3px 8px 1px;
+    border-radius: 8px;
+`
+
+const StyledFlex = styled(Flex)`
+    max-height: 83px;
+`
+
+const StyledLocalizedLink = styled(LocalizedLink)`
+    text-decoration: none;
+`
+
+// Can modify this for second phase to get the tag name
+// const getTagName = (tags) => {
+//     for (let i = 0; i < tags.length; i++){
+//         if (tags[i].tags_id.tag_name !== 'Market News') return tags[i].tags_id.tag_name
+//     }
+//     return 'Market News'
+// }
+
+const MarketNews = ({ data }) => {
+    return (
+        <SectionContainer>
+            <Container>
+                <Flex fd="column">
+                    <Header mb="20px" align="center" type="heading-3" as="h3">
+                        Market news
+                    </Header>
+                    <MarketsNewsWrapper>
+                        {data.map((data, index) => {
+                            return (
+                                <StyledLocalizedLink key={index} to={`/academy/blog/${data.slug}`}>
+                                    <StyledFlex key={index}>
```

Comment:
We don't need key for `StyledFlex`, and let's use the blog id for the `StyledLocalizedLink` key.